spec/support/pdf_spec_helper.rb

```diff
@@ -3,7 +3,8 @@ def filled_in_values(file_path)
     filled_in_fields = PdfForms.new.get_fields(file_path)
 
     filled_in_fields.each_with_object({}) do |field, hash|
-      hash[field.name] = field.value
+      # The pdf-forms gem returns HTML-escaped form values when using pdftk-java. Unescape.
```

Comment:
This general idea is alluded-to in the pdftk readme -- https://github.com/jkraemer/pdf-forms#field-names-with-html-entities -- although they're concerned about field names, not field values.

My guess is that the treatment of apostrophes is a behavior difference between `pdftk` and `pdftk-java` and the `pdf-forms` gem author doesn't know about this behavior difference.